### PR TITLE
fix issue#262

### LINKED ISSUE #262: go-sysl ints --epa option shows name of the endpoint in the diagram
### Steps to reproduce the behaviour
```
System_a:
    endpoint:
        System_b <- endpoint

System_b:
    endpoint:
      System_c <- endpoint

    endpoint_2:
      System_e <- endpoint

System_c:
    endpoint:
      System_d <- endpoint

    endpoint_2: ...

System_d:
    endpoint: ...

System_e:
    endpoint:
      System_c <- endpoint_2

Project [appfmt="%(appname)", epfmt="%(epname)", indirect_arrow_color="silver"]:

    no_passthrough:
        System_a

    passthrough_c [passthrough=["System_c"]]:
        System_a

    passthrough_d [passthrough=["System_d"]]:
        System_a

    passthrough_c_e [passthrough=["System_c", "System_e"]]:
        System_a

```
py-sysl:
`sysl ints -v -epa  -o 'py-%(epname)-epa.png' -j Project d.sysl`

go-sysl:
`ints --epa  -o 'go-%(epname)-epa.png' -j Project d.sysl`

**NOTICE**
`no_passthrough` in the image

### Expected behaviour

py-no_passthrough-epa

### Actual behaviour

go-no_passthrough-epa

### PR DESCRIPTION
**Pull request by JackLeeHal**
Fixes #262.

Changes proposed in this pull request:
- Fix issue #262 the wrong behaviour of showing epname.

@anz-bank/sysl-developers

### CODE REVIEW

File: sysl2/sysl/tests/pubsub-golden.puml [hunk 1]
```diff
@@ -22,6 +22,6 @@ state "Systema" as X_1 {
 state "Systemb" as X_2 <<highlight>> {
   state "endpoint" as _2 <<highlight>>
 }
-_2 -[#blue]> _0 : <pubsub>
-_2 -[#blue]> _1 : <pubsub>
+_2 -[#blue]> _0 : <>
```
Comment (sahejsingh) [inline sysl2/sysl/tests/pubsub-golden.puml]: extra ` : <>` does not seem to be correct.

File: sysl2/sysl/tests/epa-golden.puml [hunk 2]
```diff
@@ -24,7 +24,7 @@ state "Systemb" as X_2 <<highlight>> {
   state "endpoint" as _3 <<highlight>>
 }
 _1 -[#silver]-> _0
-_0 -[#black]> _2 : <epa needsInt>
+_0 -[#black]> _2 : < needsInt>
```
Comment (sahejsingh) [inline sysl2/sysl/tests/epa-golden.puml]: does python version also produce the same output?

File: sysl2/sysl/tests/pubsub-golden.puml (hunk 1 above)
Comment (JackLeeHal) [reply sysl2/sysl/tests/pubsub-golden.puml]: deleted

File: sysl2/sysl/tests/epa-golden.puml (hunk 2 above)
Comment (JackLeeHal) [reply sysl2/sysl/tests/epa-golden.puml]: checked, it's the same. @sahejsingh

**Comment by JackLeeHal** [comment]
@sahejsingh Review needed.

**Pull request merged**